tests/test_regex_identifier.py

```diff
@@ -39,19 +39,54 @@ def test_lat_long2():
     assert "Latitude & Longitude Coordinates" in res[0]["Regex Pattern"]["Name"]
 
 
+def test_lat_long3():
+    r = regex_identifier.RegexIdentifier()
+    res = r.check(['77\u00B0 30\' 29.9988" N'])
+    assert "Latitude & Longitude Coordinates" in res[0]["Regex Pattern"]["Name"]
+
+
+def test_lat_long4():
+    r = regex_identifier.RegexIdentifier()
+    res = r.check(["N 32\u00B0 53.733 W 096\u00B0 48.358"])
+    assert "Latitude & Longitude Coordinates" in res[0]["Regex Pattern"]["Name"]
+
+
+def test_lat_long5():
+    r = regex_identifier.RegexIdentifier()
+    res = r.check(["41\u00B024\'12.2\" N 2\u00B010\'26.5\" E"])
+    assert "Latitude & Longitude Coordinates" in res[0]["Regex Pattern"]["Name"]
+
+
+def test_lat_long6():
+    r = regex_identifier.RegexIdentifier()
+    res = r.check(["40.741895,-73.989308"])
+    assert "Latitude & Longitude Coordinates" in res[0]["Regex Pattern"]["Name"]
+
+
 def test_ip():
     r = regex_identifier.RegexIdentifier()
     res = r.check(["http://10.1.1.1"])
-    assert "Uniform Resource Locator (URL)" in res[0]["Regex Pattern"]["Name"]
+    assert "Internet Protocol (IP) Address Version 4" in res[0]["Regex Pattern"]["Name"]
```

Comment:
You can also do the reverse, assert "URL" not in res? Be aware that this:
```py
res[0]
```

only gets the first, most likely output. If you want to make sure this doesn't match on domains you'll need to check all of em :-)